### PR TITLE
WIP: ENH: datalad commands

### PR DESCRIPTION
**Pull request by bpoldrack**
Not ready yet. Just a translation of the original scripts. Needs some more work and tests.

### CODE REVIEW

File: datalad_neuroimaging/commands/create_study.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+                # Note: This is a pointer to the container we are in, if it was
+                # passed by the caller (which could be datalad-run) via
+                # SINGULARITYENV_DATALAD_CONTAINER:
+
+                add_sibling_dicts = [{
+                    'action': 'add',
+                    'name': 'github-psyinf',
+                    'url': 'https://github.com/psychoinformatics-de/cbbs-imaging-container-import.git',
+                }]
```
Comment (yarikoptic) [inline datalad_neuroimaging/commands/create_study.py]: could/shouldn't those all be parts of the configurable items to be of general applicability?

Comment (mih) [reply datalad_neuroimaging/commands/create_study.py]: Yes, agree to all, hence description says: "Not ready yet. Just a translation of the original scripts."

Comment (bpoldrack) [reply datalad_neuroimaging/commands/create_study.py]: FTR: I also agree. ;-) Apart from that, this command should eventually go away in favor of a template to be called with `datalad-create` from core anyway.

**Comment by mih** [comment]
Please merge/rebase on master.

**Comment by bpoldrack** [comment]
Update: 
I'm taking apart this PR. Current state of things go to https://github.com/psychoinformatics-de/datalad-cbbsimaging/pull/1 instead, while generalized versions will end up here. Little changes like PR #26 are separated out.

**Comment by mih** [comment]
This is now redundant with https://github.com/psychoinformatics-de/datalad-hirni

**Pull request closed without merging**